source/content/guides/composer-convert.md

```diff
@@ -35,72 +30,82 @@ export site=yoursitename
 
   ```bash{promptUser:user}
-  git checkout -b composify
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
   ```
 
-  You can replace `composify` with a branch name of your choosing, but all following steps assume this name.
-
-## Setup a Multidev (optional)
-If your Pantheon account has access to [multidev](/multidev), create a multidev to push your new code to:
-
-```bash{promptUser:user}
-git push origin composify && terminus env:create $site.dev composify
-```
+  You can replace `composerify` with another branch name. If you do, remember to adjust the other examples in this doc to match.
 
-This will setup the multidev environment to receive and demo our changed code.
+1. Copy your configuration from the default branch:
 
-## Create a New Composer Project
+  ```bash{promptUser:user}
+  git checkout master sites/default/config
+  git mv sites/default/config/* config
+  git commit -m "Pull in configuration from default branch"
+  ```
 
-1. In your local terminal, from the repository root of your Pantheon site, move a directory up:
+1. Compare your `pantheon.yml` with the new `pantheon.upstream.yml` and check for conflicts:
 
   ```bash{promptUser:user}
-  cd ..
+  git diff master:pantheon.yml pantheon.upstream.yml
   ```
 
-2. Use Composer to create a new project, using the [Pantheon Drupal 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) repository:
+1. If you need your `pantheon.yml`, copy it over (be sure to edit if needed):
 
-    ```bash{promptUser:user}
-    composer create-project pantheon-systems/example-drops-8-composer $site-composer
-    cd $site-composer
-    ```
+  ```bash{promptUser:user}
+  git checkout master pantheon.yml
+  git add pantheon.yml
+  git commit -m 'Copy my pantheon.yml'
+  ```
 
-This will create a new directory based on the example project [pantheon-systems/example-drops-8-composer](https://github.com/pantheon-systems/example-drops-8-composer) in the `$site-composer` directory.
+### (Optional) Return to Drupal 8
 
-### Copy pantheon.upstream.yml
-Since the drops-8 upstream has a `pantheon.upstream.yml` and the example-drops-8-composer upstream does not, copy over our old file for the platform to properly load the site. From the `$site-composer` directory, run:
+If you’re not ready to update to Drupal 9 yet. (Advisable: test the Composer conversion before testing the Drupal 9 upgrade)
+Note that you might want to hand-edit your composer.json and remove the drupal/core entry altogether. The empty drupal/core entry from the steps above will cause drupal/core to be erased and redownloaded frequently
 
 ```bash{promptUser:user}
-cp ../$site/pantheon.upstream.yml .
+cd upstream-configuration/
+composer remove zaporylie/composer-drupal-optimizations --no-update
+composer require drupal/core-recommended:^8 --no-update
+cd -
+composer config --json "extra.patches.drupal/core" '{}'
+composer update
```

Comment:
```suggestion
# Remove new MariaDB directive from pantheon.upstream.yml
sed -i 's/^\s*version: 10.[0-9]*$/  {}/' pantheon.upstream.yml
composer update
```
There's a database version directive in the `pantheon.upstream.yml` that, at present, must be removed for any existing site regardless of Drupal version. I'm including it in this "Optional return to Drupal 8" section though to future-proof this documentation.

Reply:
@EdwardAngert @greg-1-anderson I thought I'd been down this road before. This is how we would want to deal with https://github.com/pantheon-systems/documentation/pull/6145#discussion_r575312499 I believe. Removing the database version will allow us to upgrade the sites as time goes on, so it's better than including the database version and requiring 10.0.

Reply:
Doh, bad idea to modify pantheon.upstream.yml. I've tested however that if you put this into `pantheon.yml` instead, then you override `pantheon.upstream.yml`'s 10.4, and also maintain the desired effect of the database version not being strictly pinned:
```
database:
  {}
 ```
That's what I think I'll have the terminus conversion tool I'm working on do if the user says they're staying with Drupal 8.

Reply:
I think that this doc is blocked on the database upgrade, so we can just leave this step out.